opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -372,8 +372,7 @@ def _mapping_to_file(self, port, mapping, fixed_ips):
         else:
             if (mapping.get('enable_dhcp_optimization', False) and
                'subnets' in mapping):
-                    self._map_dhcp_info(fixed_ips, mapping,
-                                        mapping_dict)
+                self._map_dhcp_info(fixed_ips, mapping, mapping_dict)
```

Comment:
Interesting that PEP8 doesn't flag this - I though the line above would have needed the extra 4 spaces?

Reply:
It actually complains about the extra spaces. I've had to remove them here and several other places in this repo and the GBP repo.

Reply:
Ah - good to know. Thanks!